rules/css-changes/rule.md

```diff
@@ -0,0 +1,104 @@
+---
+type: rule
+title: Do you consider factors before making a CSS change?
+seoDescription: This rule explains when to involve a developer for CSS changes and when to make small changes yourself
+uri: css-changes
+authors:
+  - title: Betty Bondoc
+    url: https://www.ssw.com.au/people/betty-bondoc/
+  - title: Tiago Araujo
+    url: https://www.ssw.com.au/people/tiago-araujo/
+created: 2024-10-30T14:59:00.000Z
+guid: 3a3f7ec5-8360-43db-ba6b-1dcdc7c633e6
+---
+Making CSS changes can range from straightforward tweaks to complex adjustments that impact multiple elements or pages. Knowing when to handle changes yourself and when to seek a developer's input can save time and reduce risks.
+
+<!--endintro-->
+
+## When to make CSS changes yourself
+
+For small or simple adjustments, it’s often faster and more efficient to handle the change yourself. This applies when:
+
+* **The change is isolated**: Adjustments affect only a specific element or page
+* **Minimal complexity**: Minor visual changes, such as color or spacing tweaks
+* **No significant dependencies**: No impact on other components or functionalities
+* **Low testing requirements**: The changes don’t need to be tested across multiple browsers or devices
+
+
+::: good  
+```css
+.button {
+  font-size: 16px;
```

Comment:
Hey @bettybondoc 

- [ ] 1. Please make this example using `rem` which is a best practice